.github/workflows/wheels.yml

```diff
@@ -0,0 +1,180 @@
+# Workflow to build wheels for upload to PyPI.
+# Inspired by numpy's cibuildwheel config https://github.com/numpy/numpy/blob/main/.github/workflows/wheels.yml
+#
+# In an attempt to save CI resources, wheel builds do
+# not run on each push but only weekly and for releases.
+# Wheel builds can be triggered from the Actions page
+# (if you have the perms) on a commit to master.
+#
+# Alternatively, you can add labels to the pull request in order to trigger wheel
+# builds.
+# The label(s) that trigger builds are:
+# - Build
+name: Wheel builder
+
+on:
+  schedule:
+  #        ┌───────────── minute (0 - 59)
+  #        │  ┌───────────── hour (0 - 23)
+  #        │  │ ┌───────────── day of the month (1 - 31)
+  #        │  │ │ ┌───────────── month (1 - 12 or JAN-DEC)
+  #        │  │ │ │ ┌───────────── day of the week (0 - 6 or SUN-SAT)
+  #        │  │ │ │ │
+  - cron: "42 1 * * 4"

```

Comment:
Should we disable the cron for now while there isn't regular use for the wheels?

Reply:
It would be nice to have it to verify the general workflow for uploading wheels all the way to anaconda works with this.